backend/src/api/database-migration.ts

```diff
@@ -1123,6 +1128,23 @@ class DatabaseMigration {
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;`;
   }
 
+  private getCreateAccelerationsTableQuery(): string {
+    return `CREATE TABLE IF NOT EXISTS accelerations (
+      txid varchar(65) NOT NULL,
+      added date NOT NULL,
```

Comment:
Should the `date` value not be more specific than down to the day?

Reply:
yes, it should be a datetime. I'll fix that.